plugins/win-capture/get-graphics-offsets/CMakeLists.txt

```diff
@@ -4,8 +4,6 @@ add_library(_get-graphics-offsets INTERFACE)
 target_sources(_get-graphics-offsets INTERFACE d3d8-offsets.cpp d3d9-offsets.cpp dxgi-offsets.cpp
                                                get-graphics-offsets.c get-graphics-offsets.h)
 
-target_compile_options(_get-graphics-offsets INTERFACE $<IF:$<CONFIG:Debug>,/MTd,/MT>)
```

Comment:
No longer needed?

Reply:
At least I've encountered no compile errors in Debug or Release without it in Visual Studio 17.

Reply:
This is for ensuring that the standalone program does not depend on the dynamic CRT. I'm not entirely sure why we do that with our standalone helper programs given OBS itself requires the same CRT.

Reply:
Got it, in that case the "cleaner" approach is to set the `MSVC_RUNTIME_LIBRARY` target property to `"MultiThreaded$<$<CONFIG:Debug>:Debug>"` (which is the officially suggested way since CMake 3.5).